svo (curing skeleton, controllers, action system).xml

```diff
@@ -2264,6 +2263,35 @@ function svo.sendcuring(what)
   sk.sendqueuecmd(what)
 end
 
+function svo.sendcuringprio(prio, iter, type)
+  prio = " "..prio
+  if type == 'affliction' then
+    table.insert(svo.sk.affqueue, prio)
+    if #sk.affqueue >= 20 then
+      if #sk.sendqueue >= 6 then
+        sk.dosendqueue()
+      end
+      sk.sendqueue[#sk.sendqueue+1] = "curing priority"..table.concat(svo.sk.affqueue)
+      if not sk.sendcuringtimer then
+        sk.sendcuringtimer = tempTimer(0, sk.dosendqueue)
+      end
+      sk.affqueue = {}
+    end
+  elseif type == 'defence' then
+    table.insert(svo.sk.defqueue, prio)
+    if #sk.defqueue >= 20 then
+      if #sk.sendqueue >= 6 then
+        sk.dosendqueue()
+      end
+      sk.sendqueue[#sk.sendqueue+1] = "curing priority defence"..table.concat(svo.sk.defqueue)
+      if not sk.sendcuringtimer then
+        sk.sendcuringtimer = tempTimer(0, sk.dosendqueue)
+      end
+      sk.defqueue = {}
+    end
```

Comment:
I'd put an else here in the end - so an error doesn't get swallowed if you typo affliction/defence.

Reply:
Done